client/python/cryoet_data_portal/src/cryoet_data_portal/data/schema.graphql

```diff
@@ -1194,7 +1194,7 @@ type annotations {
   """
   object_description: String
 
-  """Gene Ontology Cellular Component identifier for the annotation object"""
```

Comment:
Are these auto-generated from another source? (Hasura?) May be nice to have a BE person on the PR as well?

Reply:
@jgadling, Do you know? Thanks!

Reply:
Yep, this file is read from the hasura api schema. In order for this change to stick, we'll need a PR for the backend repo along these lines:
https://github.com/chanzuckerberg/cryoet-data-portal-backend/blob/main/api_server/migrations/cryoetdataportal/1722635643562_alter_comments/up.sql

Reply:
Sounds good, thanks! I'll go ahead and revert these changes then.